### PR TITLE
Remove LDAP support

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

**Review by amandahla — DISMISSED** [review]
LGTM 
(I approved but then saw that there are missing files to ignore the license header, I'll approve again after this change).

**Review by amandahla — DISMISSED** [review]
LGTM

**Review by gtrkiller — DISMISSED** [review]
LGTM

File: load_tests/load-test.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const options = {
   vus: 50,
   thresholds: {
     http_req_failed: ['rate<0.01'], // http errors should be less than 1%
-    http_req_duration: ['p(95)<1500'], // 95 percent of response times must be below 1500ms
+    http_req_duration: ['p(90)<1500'], // 90 percent of response times must be below 1500ms
```
Comment (merkata) [inline load_tests/load-test.js:12]: :)

**Review by merkata — APPROVED** [review]
Looks good!

**Review by amandahla — APPROVED** [review]
LGTM

**Pull request merged**